### PR TITLE
Updating Kikoff guide to mention upcoming change to format of data we send them

### PR DESCRIPTION
**Pull request by liamhalemccarty**
[Ticket]()

## Checklist

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas _to a borderline excessive amount_
- [ ] I have made any relevant corresponding changes to the documentation including the project readme.
- [ ] I have run and tested the changes locally
- [ ] If it is a core feature, I have added an appropriate amount of unit tests.
- [ ] Any dependent changes have been merged and published in upstream projects

### CODE REVIEW

File: docs/kikoff-guide.mdx [hunk 1]
```diff
@@ -612,13 +612,12 @@ The 1-Click endpoint is [`POST /1-click`](/api-reference#1-click). Call it to be
...
 ```typescript title="POST /1-click Request Body: Recommendation for Kikoff"
 {
-    "phone": string, // 10-digit string with +1 country code at the beginning
-    "email": string,
+    "phone": string // 10-digit string with +1 country code at the beginning
 }
 ```
 
 :::note[Why this request body?]
-- `phone` and `email` allow us to source verified data about the user and send it to you.
```
Comment (rsmets) [inline docs/kikoff-guide.mdx:621]: When did we decide to drop `email`?

File: docs/kikoff-guide.mdx [hunk 2]
```diff
@@ -697,6 +696,215 @@ One particularly valuable piece of metadata is **`verificationMethod`**, which d
...
+            "identifiers": {
+                "verificationMethod": {
+                    "phone": "phone_carrier"
+                }
+            },
+            "credentials": {
+                "id": {
+                    "email": "8a1d4e35-413d-496b-b499-8810b55cfb5c",
+                    "fullName": {
+                        "firstName": "2e6a7b9a-e93e-43ba-98a9-c554f4e16457",
```
Comment (rsmets) [inline docs/kikoff-guide.mdx:808]: Hmm I really wonder why an Acceptor would care about the a credential `id` 🤔. I suppose in the spirt of returning everything we already were this is the correct way to go.

**Review by rsmets — COMMENTED** [review]
Some in line questions. Also, should we update the general guide as well or just wait until the implementation is all done?

File: docs/kikoff-guide.mdx (hunk 1 above)
Comment (liamhalemccarty) [reply docs/kikoff-guide.mdx:621]: https://verified-inc.slack.com/archives/CMHP6C3MY/p1706057214499859?thread_ts=1706053723.975469&cid=CMHP6C3MY

File: docs/kikoff-guide.mdx (hunk 2 above)
Comment (liamhalemccarty) [reply docs/kikoff-guide.mdx:808]: Yeah...

**Comment by liamhalemccarty** [comment]
> Some in line questions. Also, should we update the general guide as well or just wait until the implementation is all done?

I think we should wait until the implementation is done, to avoid causing too much confusion

**Review by rsmets — APPROVED** [review]

**Pull request merged**